### ISSUE TITLE
Neovide changes it's width on each start.

### ISSUE THREAD

**Issue by xbladesub** [neovide/neovide#1521]
```lua
-- vim.g.neovide_transparency = 0.9
vim.g.neovide_cursor_vfx_mode = "ripple"
vim.g.neovide_remember_window_size = true
vim.g.neovide_remember_window_position = true
vim.g.neovide_cursor_antialiasing = true
-- vim.g.neovide_fullscreen = true
```

Neovide started to increase it's width on each start:
{"window":{"Windowed":{"position":{"x":0,"y":76},"size":{"width":400,"height":59}}}}
{"window":{"Windowed":{"position":{"x":0,"y":76},"size":{"width":800,"height":59}}}}
and so on until it crash...

after that it crashes on start if I not remove neovide-settings.json file.

I run it from shell script like that:
```sh
 #!/bin/sh
export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"$HOME/.local/share/lunarvim"}"
export LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-"$HOME/.config/lvim"}"
export LUNARVIM_CACHE_DIR="${LUNARVIM_CACHE_DIR:-"$HOME/.cache/lvim"}"

cd $HOME
exec /opt/homebrew/bin/neovide --frame=none --multigrid -- -u "$LUNARVIM_RUNTIME_DIR/lvim/init.lua" "$@"

```
Everything was working fine util today. Same issue on both MacBook and Mac mini
Any suggestions how to fix that?

**Comment by yngwi** [comment]
Sounds like the issue I reported some time ago: https://github.com/neovide/neovide/issues/1400

**Comment by fredizzimo** [comment]
The problem is fixed in Neovide 0.11.0, the window size is saved as pixels instead of columns and rows.
